README.md

```diff
@@ -25,8 +25,8 @@ var mediator = require('fh-wfm-mediator');
 var Topics = require('fh-wfm-mediator/lib/topics');
 
 var topics = new Topics(mediator)
-  .prefix('wfm')
-  .entity('user')
+  .withPrefix('wfm')
+  .withEntity('user')
```

Comment:
Actually as part of the chained calls, this one does have semi! And some of the other examples were missing closing parens... Would be nice to have a linter that would go inside github-flavored md snippets, or support for including snippets.